### PR TITLE
task 1 again

### PR DESCRIPTION
**Pull request by yvechar**

### CODE REVIEW

**Review by AlexeiRomankov — COMMENTED** [review]
Commits on Mar 19, 2019
Deadline=03/10/2019 18-00
?

File: task-11/task-11/Program.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+namespace Task_1
+{
+    /// <summary>
+    /// The Class EntryPoint provides main
+    /// </summary>
+    /// <param name="args">arguments from command line</param>
+    class EntryPoint
+    {
+        static void Main(string[] args)
```
Comment (AlexeiRomankov) [inline task-11/task-11/Program.cs:11]: please add summary for the method

File: task-11/task-11/Program.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            try
+            {
+                foreach (string arg in args)
+                {
+                    SearchForConSymbols Obj = new SearchForConSymbols();
+                    Obj.SearchAndDisplay(arg);
+                }
+            }
+            catch (FormatException)
+
```
Comment (AlexeiRomankov) [inline task-11/task-11/Program.cs:22]: please delete blank line

File: task-11/task-11/Program.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    /// <param name="args">arguments from command line</param>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                foreach (string arg in args)
+                {
+                    SearchForConSymbols Obj = new SearchForConSymbols();
```
Comment (AlexeiRomankov) [inline task-11/task-11/Program.cs:17]: unclear name of the vvariable

File: task-11/task-11/Program.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    /// The Class EntryPoint provides main
+    /// </summary>
+    /// <param name="args">arguments from command line</param>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                foreach (string arg in args)
```
Comment (AlexeiRomankov) [inline task-11/task-11/Program.cs:15]: do you really need all arguments? the task says in other way

Comment (AlexeiRomankov) [inline task-11/task-11/Program.cs:15]: Is it better to check that arguments is not presented?

File: task-11/task-11/Program.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                    SearchForConSymbols Obj = new SearchForConSymbols();
+                    Obj.SearchAndDisplay(arg);
+                }
+            }
+            catch (FormatException)
+
+            {
+                Console.WriteLine("Empty String");
+            }
+
```
Comment (AlexeiRomankov) [inline task-11/task-11/Program.cs:26]: please delete blank line

File: task-11/task-11/SearchForConSymbols.cs [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    internal class SearchForConSymbols
+    {
+        public String NewLine;
+        public SearchForConSymbols() { }
+        /// <summary>
+        /// The Constructor that checks input string
+        /// </summary>
+        public SearchForConSymbols(string arg)
+        {
+
```
Comment (AlexeiRomankov) [inline task-11/task-11/SearchForConSymbols.cs:20]: please delete blank line

File: task-11/task-11/SearchForConSymbols.cs [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+{
+    /// <summary>
+    /// Class  SearchAndDisplay is searching repeating symbols and than display them
+    /// It also checks the input string and if it has less than 2 symbols throws the exception
+    /// </summary>
+    internal class SearchForConSymbols
+    {
+        public String NewLine;
+        public SearchForConSymbols() { }
+        /// <summary>
```
Comment (AlexeiRomankov) [inline task-11/task-11/SearchForConSymbols.cs:15]: please add blank line before summary

File: task-11/task-11/SearchForConSymbols.cs [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+namespace Task_1
+{
+    /// <summary>
+    /// Class  SearchAndDisplay is searching repeating symbols and than display them
+    /// It also checks the input string and if it has less than 2 symbols throws the exception
+    /// </summary>
+    internal class SearchForConSymbols
+    {
+        public String NewLine;
+        public SearchForConSymbols() { }
```
Comment (AlexeiRomankov) [inline task-11/task-11/SearchForConSymbols.cs:14]: please add summary for the construct

File: task-11/task-11/SearchForConSymbols.cs [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+                {
+                    NewLine += args[i + 1 + j];
+                    if (args[i + j] != args[i + 1 + j])
+                    {
+                        Console.WriteLine(NewLine);
+                    }
+                    else break;
+                }
+            }
+
```
Comment (AlexeiRomankov) [inline task-11/task-11/SearchForConSymbols.cs:51]: please delete blank line

File: task-11/task-11/SearchForConSymbols.cs [hunk 10]
```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+
```
Comment (AlexeiRomankov) [inline task-11/task-11/SearchForConSymbols.cs:3]: please delete blank lines except one

File: task-11/task-11/Program.cs (hunk 1 above)
Comment (yvechar) [reply task-11/task-11/Program.cs:11]: i have some problems with repository. So IgorKheidorov said to creat new one and add files there. it is about  Commits on Mar 19, 2019
Deadline=03/10/2019 18-00
?

**Pull request merged**